src/QuadForm/Torsion.jl

```diff
@@ -2159,3 +2160,81 @@ underlying abelian group is in Smith normal form.
 """
 is_snf(T::TorQuadModule) = is_snf(abelian_group(T))
 
+################################################################################
+#
+#  Isotropic check
+#
+################################################################################
+
+@doc raw"""
+    is_totally_isotorpic(T::TorQuadModule)
+
+Return whether the quadratic form on the torsion quadratic module `T` vanishes.
+"""
+function is_totally_isotropic(T::TorQuadModule)
```

Comment:
This should use that the inner product is symmetric, so that you only need
n(n+1) computations instead of n^2